runtime/itests/reference-mappings/README.md

```diff
@@ -0,0 +1,98 @@
+# Reference Mappings Overview #
+
+## Introduction ##
+
+The itests/reference-mappings test cases were created with the intent of testing a competely exhaustive (CE) combination of field types and data formats.
+
+## Test Scenarios ##
+
+
+### Format to Format Testing ###
+
+The tests are broken up by high-level functional use cases based on source and target format
+
+For example:
+
+1. Java to Java, Json and Xml mappings
+2. Json to Java, Json and Xml mappings
+3. Xml to Java, Json and Xml mappings
+4. multidoc mappings
+
+### Test artifacts ###
+
+Within each of these high-level scenarios, the use cases are broken down into sub-test types. Each test scenario includes:
+
+1. A complete atlasmapping file
+2. Sample generated source document
+3. Validation routine to confirm the expected output is correct
+
+Note: For simplicity and the ability to re-use validation methods, many of the test objects share the same name and data shape
+
+### Data Type Coverage ###
+
+The sub-tests include coverage for:
+
+1. Primitive data types (int, long, short, etc)
+2. Boxed Primitive data types (Integer, Long, Short, etc)
+3. Complex data types (Order, Customer, etc)
+4. Repeating primitive types (array, list, etc)
+5. Repeating complex types
+6. Rooted and un-rooted Json and Xml object types
+7. Fully qualified namespaced Xml and non-namespaced Xml documents
```

Comment:
Can you add file name rules of test mapping files and test class corresponding to this list?